bitly_api/bitly_api.py

```diff
@@ -323,9 +329,17 @@ def user_tracking_domain_shorten_counts(self, domain, **kwargs):
             "v3/user/tracking_domain_shorten_counts", params, **kwargs)
         return data["tracking_domain_shorten_counts"]
 
-    def user_info(self, **kwargs):
+    def user_info(self, login=None,full_name=None,**kwargs):
         """return or update info about a user"""
-        data = self._call_oauth2("v3/user/info", kwargs)
```

Comment:
The previous call should correctly pass `login` and `full_name` along when called with `.user_info(login=....)`. Were you encountering a problem with that?

This change is problematic because it switches to `_call_oauth2_metrics` instead of continuing to use `_call_oauth2`. Can you revert this change? If you'd like to list `login` and `full_name` in the signature, the right way to do that is `kwargs['login'] = login` etc.